configs/nvme-storage-configurator/base/daemonset.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nvme-storage-configurator
+  labels:
+    app: nvme-storage-configurator
+spec:
+  selector:
+    matchLabels:
+      app: nvme-storage-configurator
+  template:
+    metadata:
+      labels:
+        app: nvme-storage-configurator
+    spec:
+      nodeSelector:
+        konflux-ci.dev/storage: "nvme"
+      hostPID: true
+      serviceAccount: nvme-storage-configurator
+      initContainers:
+        - name: configure
+          image: "quay.io/openshift/origin-network-tools"
+          command: ['/bin/bash', '-c', '/bin/bash -x /scripts/configure.sh']
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: /scripts
+              name: script-config
+            - mountPath: /host-etc-systemd-dir
+              name: etc-systemd-dir
+            - mountPath: /host-usr-local-bin
+              name: usr-local-bin
+            - mountPath: /host-dev-dir
+              name: dev-dir
+      containers:
+        - name: pause
+          image: registry.redhat.io/rhel8/pause:8.10
+      volumes:
+        - name: etc-systemd-dir
+          hostPath:
+            path: /etc/systemd/system
+        - name: usr-local-bin
+          hostPath:
+            path: /usr/local/bin
+        - name: dev-dir
```

Comment:
why this mount is needed? when running a privileged pod does it get access to the hosts /dev dir?

Reply:
We're making use of hostPaths in order to access the file system of ec2 instance. We have mapped `/dev` to programatically the nvme device name rather than hard coding `nvme1n1`.

Reply:
when running a privileged container, its /dev directory should be the same as the host, making this mount redundant.

Reply:
I have no idea about this. But when we tried running this without mouting the /dev directory, we couldn't access the /dev/xxxx. I assumed, we couln't not be accessed because it was not mounted.